### PR TITLE
Location search styling

### LINKED ISSUE #219: As Renter, I want the location search list to be displayed as floating cards so that the list looks nicer.
**User story**

As Renter, I want the location search list to be displayed as floating cards so that the list looks nicer.

**Acceptance criteria**

**Definition of Done**

I will know this is done when a couple of things have been done. Each location should display as a floating card. The card should display a small thumbnail of the location (if it has an available picture), a bold number that signifies how many apartments in that location match the search criteria, and maybe a bit of text identifying the location.  

The card grid should be paginated, with enough cards displayed on each page that it feels full but not crammed (this will depend on the size of the cards). 

(There is a django module for pagination)

### PR DESCRIPTION
**Pull request by mok232**
This PR closes #219 

- Displays search results (locations) as cards
- 2-column format for styling
- Displays the number of matching apartments that satisfy the given criteria at that location
- Adds pagination (2 locations displayed per page)

### CODE REVIEW

**Comment by ab7289** [comment]
Would it be possible to shrink the cards so that we could get 2 rows of 3 columns per page (total of 6 per page)? I just feel like if there are a lot of search results, say even just 10, it would be annoying to have to click through so many pages with only two displayed per page.

Would it also be possible to give each card a small box-shadow so that it looks like its floating a bit off the page?

File: search/tests.py [hunk 1]
```diff
@@ -193,6 +195,40 @@ def test_search_page_all_params(self):
...
+    #         suite_num=suite_num,
+    #         number_of_bed=number_of_bed,
+    #         image=image,
+    #         rent_price=rent_price,
+    #         location=loc,
+    #     )
+    #     apt.save()
+    #
+    #     response = self.client.get(
+    #         "/search/?query=11218&min_price=500&max_price=2000&bed_num=4"
```
Comment (ab7289) [inline search/tests.py]: you may need to also mock the `normalize_us_address` function so it returns the search terms that you want (and also doesn't make a call to geocode. you can use `@mock.patch("search.views.normalize_us_address")`. For reference you can take a look at lines 224-246 in `search/tests.py` in my branch feature/apa-rented

Comment (mok232) [reply search/tests.py]: Thank you for the reference! It worked. Also, changed the formatting to 2 rows and 3 cols and added a box shadow.

Comment (ab7289) [reply search/tests.py]: excellent, looks much more user friendly now

**Pull request merged**